Shared/API/CloudFunction.swift

```diff
@@ -11,11 +11,66 @@ import Parse
 import Combine
 
 protocol CloudFunction: StatusableRequest {
-    func makeRequest(andUpdate statusables: [Statusable], viewsToIgnore: [UIView]) -> AnyPublisher<ReturnType, Error>
+    func makeRequest(andUpdate statusables: [Statusable],
+                     viewsToIgnore: [UIView]) -> AnyPublisher<ReturnType, Error>
+    func makeAsyncRequest(andUpdate statusables: [Statusable],
+                          viewsToIgnore: [UIView]) async throws -> ReturnType
```

Comment:
I'm converting all of the cloud functions to be async. I'm doing it in stages so I'll have both a sync'd and async function for now.